libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -132,6 +246,69 @@ def add_texts(
         # Insert the embedded texts in the vector store table.
         return self._insert_embeddings(texts, embedded_texts, metadatas, ids)
```

Comment:
We should restrict addition of texts that dont have their embeddings length = to the embeddings_length property.

Reply:
I'm not sure I follow here. We already have a default value for embedding_length in the constructor.

Reply:
As it is right now, embeddings with embeddings length > embeddings_length property will fail, thus adding that text is restricted. But embeddings with length <= embeddings_length property will be inserted.

Reply:
We need to let only that text which exactly matches the embeddings_length to get inserted